apps/backend/src/modules/points-of-interest/points-of-interest.service.ts

```diff
@@ -50,13 +50,13 @@ class PointsOfInterestService implements Service {
 	}
 
 	public async findAll(
-		query: null | PointsOfInterestSearchQuery,
+		query: null | PointsOfInterestFindAllOptions,
```

Comment:
let's rename it to options now
```suggestion
		options: null | PointsOfInterestFindAllOptions,
```